### PR TITLE
Consolidated README

### PR DESCRIPTION
**Pull request by joll90**
Consolidated README in 50_Server and added README to top level directory.

### CODE REVIEW

**Review by scarter93 — CHANGES_REQUESTED** [review]

File: 50_Server/README.md [hunk 1]
```diff
@@ -1,19 +1,79 @@
...
+    cd citrineos-core/50_Server
+    ```
+
+1. Install project dependencies:
+
+   ```shell
+   ./unix-init-install-all.sh
+   ```
+
+1. Start the server and it's supporting infrastructure with:
```
Comment (thanaParis) [inline 50_Server/README.md:35]: it's -> its

File: 50_Server/README.md [hunk 2]
```diff
@@ -1,19 +1,79 @@
...
+
+### Configuration
+
+CitrineOS requires configuration to allow your OCPP 2.0.1 compliant charging stations to connect. 
+
+We recommend running and developing the project with docker-compose set-up.
+
+However if you like to rather run it locally and need to adjust where the server is connecting to, please locally (only) adjust the configuration file at `50_Server/src/config/envs/local.ts`
+
+### Starting the Server
```
Comment (thanaParis) [inline 50_Server/README.md:49]: Can we specify "Starting the Server without Docker configuration"

File: README.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+This is the main part of CitrineOS containing the actual charging station management logic, OCPP message routing and all modules.
+
+All documentation and the issue tracking can be found in our main repository here: https://github.com/citrineos/citrineos.
+
+## Overview
+
+CitrineOS is developed in TypeScript and runs on `NodeJS`.
+
+The system features:
```
Comment (thanaParis) [inline README.md:11]: I think we should mention it uses the fastify web framework

File: README.md [hunk 4]
```diff
@@ -0,0 +1,136 @@
...
+
+### Installation
+
+1. Clone the CitrineOS repository to your local machine:
+
+    ```shell
+    git clone https://github.com/citrineos/citrineos-core
+    ```
+
+1. Navigate to the CitrineOS Server directory:
```
Comment (thanaParis) [inline README.md:44]: All of these are numbered 1

File: README.md [hunk 5]
```diff
@@ -0,0 +1,136 @@
...
+
+### Configuration
+
+CitrineOS requires configuration to allow your OCPP 2.0.1 compliant charging stations to connect. 
+
+We recommend running and developing the project with docker-compose set-up.
+
+Starting the Server without Docker configuration is also possible. To change necessary configuration for local execution, please adjust the configuration file at `50_Server/src/config/envs/local.ts`. Make sure any changes to the local configuration do not make it into your PR.
+
+### Starting the Server
```
Comment (thanaParis) [inline README.md:70]: We don't want someone to jump to this section of the README and think this is the preferred way to start the server (that would be docker-compose up)
Please make this "Starting the Server without Docker configuration"

**Review by scarter93 — APPROVED** [review]

**Pull request merged**